src/methods/scgpt_zero_shot/config.vsh.yaml

```diff
@@ -0,0 +1,39 @@
+__merge__: ../../api/comp_method.yaml
+
+name: scgpt_zero_shot
+label: scGPT zero-shot
+summary: "Reference mapping using cell embedding by pretrained scGPT model."
+description: |
+  scGPT is a foundation model for single-cell biology based on a generative pre-trained transformer and trained on a repository of over 33 million cells. Following the zero-shot approach, a pre-trained scGPT model is used to embed cells and map unlabelled cells in a query set to the reference dataset with provided annotations. 
+
+references: 
+  doi: 10.1038/s41592-024-02201-0
+
+links: 
+  documentation: https://scgpt.readthedocs.io/en/latest/
+  repository: https://github.com/bowang-lab/scGPT
+
+info:
+  preferred_normalization: log_cp10k
+
+resources:
+  - type: python_script
+    path: script.py
+
+engines:
+  - type: docker
+    image: openproblems/base_pytorch_nvidia:1.0.0
+    setup:
+      - type: python
+        pypi:
```

Comment:
Did you try installing **faiss**? I can't remember if I couldn't get it to work or I it just wasn't needed for other tasks but the tutorial says things will be faster if you can install it.

Reply:
Haven't done it yet but will give it a try.